src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -141,6 +141,24 @@ public void cancelDispute(@PathVariable Integer id) {
 	public void submitDispute(@PathVariable Integer id) {
 		disputeService.setStatus(id, DisputeStatus.PROCESSING);
 	}
+	
+	/**
+	 * PUT endpoint that updates the dispute detail, setting the new value for the fields passed in the body.
+	 *
+	 * @param dispute to be updated
+	 * @param id of the saved {@link Dispute} to update
+	 * @return updated {@link Dispute}
+	 */
+	@Operation(summary = "Updates the properties of a particular Dispute record based on the given values.")
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "Ok"),
+		@ApiResponse(responseCode = "400", description = "Bad Request."),
+		@ApiResponse(responseCode = "404", description = "Dispute record not found. Update failed.")
+	})
```

Comment:
I think 405 should also be returned if there are validation errors in the submitted Dispute.  Ie, the rejectedReason can only be 256 characters long.  Currently if I try to update a Dispute and put a rejectedReason with > 256 this method returns a 500 error, not 405 error.

Reply:
We can add this later in a separate PR.  I noticed the POST also doesn't do this but should.

Reply:
That's true. Yes, maybe we can create a separate PR for this.